ITR/temperature_score.py

```diff
@@ -172,10 +172,16 @@ def _calculate_company_score(self, data):
 
         # Calculate the GHC--using umean to deal with uncertainties
         # FIXME: what about median vs. mean?
-        company_data = umean(data[
-            [self.c.COLS.COMPANY_ID, self.c.COLS.TIME_FRAME, self.c.COLS.SCOPE, self.c.COLS.GHG_SCOPE12,
-             self.c.COLS.GHG_SCOPE3, self.c.COLS.TEMPERATURE_SCORE, self.c.SCORE_RESULT_TYPE]
-        ].groupby([self.c.COLS.COMPANY_ID, self.c.COLS.TIME_FRAME, self.c.COLS.SCOPE]))
+        if ITR.HAS_UNCERTAINTIES:
+            company_data = umean(data[
+                [self.c.COLS.COMPANY_ID, self.c.COLS.TIME_FRAME, self.c.COLS.SCOPE, self.c.COLS.GHG_SCOPE12,
+                 self.c.COLS.GHG_SCOPE3, self.c.COLS.TEMPERATURE_SCORE, self.c.SCORE_RESULT_TYPE]
+            ].groupby([self.c.COLS.COMPANY_ID, self.c.COLS.TIME_FRAME, self.c.COLS.SCOPE]))
+        else:
+            company_data = data[
+                [self.c.COLS.COMPANY_ID, self.c.COLS.TIME_FRAME, self.c.COLS.SCOPE, self.c.COLS.GHG_SCOPE12,
+                 self.c.COLS.GHG_SCOPE3, self.c.COLS.TEMPERATURE_SCORE, self.c.SCORE_RESULT_TYPE]
+            ].groupby([self.c.COLS.COMPANY_ID, self.c.COLS.TIME_FRAME, self.c.COLS.SCOPE]).mean()
```

Comment:
we have a wide copy-paste here, maybe we could optimize

Reply:
Yes, it's all about deciding how much we want to make `mean` a polymorphic function across Float64 and UFloat.  I didn't want to think too hard about it, but welcome suggestions.